### ISSUE TITLE
Does not work anymore (reason unknown)

### ISSUE THREAD

**Issue by pixelass** [ionutvmi/sublime-jsfmt#30]
With the latest update the plugin stopped working for me. It does not matter if I save with autoformat or use try to use the commands (naming also changed? ALL UPPERCASE)

<img width="407" alt="screen shot 2015-09-09 at 4 51 44 pm" src="https://cloud.githubusercontent.com/assets/1148334/9765195/4e29fb54-5713-11e5-9ba4-5893186518be.png">

If you need more info just ask

**Comment by ionutvmi** [comment]
Do you get any errors in the console ? (you can open that by pressing ``Ctrl + ` ``)
Also can you try to uninstall and install again ?

If it's not fixed please provide more info about your environment (OS, ST version).

**Comment by pixelass** [comment]
no errors in console.
ST: Build 3083
OS: OS X 10.10.4

**Comment by ionutvmi** [comment]
what is your nodejs installed version ?  `node -v`

**Comment by pixelass** [comment]
v2.3.0

**Comment by ionutvmi** [comment]
Is that iojs ?
Can you try installing the latest stable version or nodejs https://nodejs.org/en/ ?

**Comment by royriojas** [comment]
Same here. I use node installed with `nvm (0.12.0)` if that matters

**Comment by royriojas** [comment]
Sadly uninstalling/installing didn't fix the issue, and console shows no errors

**Comment by royriojas** [comment]
It is actually weird, I uninstalled and installed it again and the `user settings` were still the old `user settings`. It seems that if there were some plugins there, the update remove them, installing the plugins again made things to work again for me, but still errors were not shown

**Comment by pixelass** [comment]
@royriojas is it working for you now? what did you change?

**Comment by royriojas** [comment]
hi @pixelass 

1. unistalled jsfmt
2. reinstalled jsfmt
3. I notice that my old jsfmt user settings was there.
4. I just reinstall the plugins that were listed in my config file. `esformatter-jsx`, `esformatter-quotes`, etc... following this guide: https://github.com/royriojas/esformatter-jsx/wiki/Usage-with-jsfmt

And run the command from inside sublime and it was working again

**Comment by ionutvmi** [comment]
I have downgraded the get-stdin package, it should now work as before in the 1.0.9 release
It should be in the package control in a few hours.

@pixelass can you please check if everything works for you ?

**Comment by royriojas** [comment]
@ionutvmi Yesterday it stop working for me again, and after redo the steps I did before I also have to modify the path to nodejs.

I guess that something changed in the code in the way `jsfmt` is detecting the node binary in osx. Since I installed node using [nvm](https://github.com/creationix/nvm) the node version in my current path was different from the one detected by `jsfmt`. And it fails trying to use the Global Promise not available in the version of node detected. Probably downgrading `get-stdin` package will fix this issue, but in the meantime using a version of node that include the global Promise object will help as well (like `node v0.12.0`)
